app/scripts/models/sync-engines.js

```diff
@@ -60,7 +60,7 @@ define((require, exports, module) => {
       text: t('Preferences')
     },
     {
-      checked: false,
+      checked: true,
       id: 'addresses',
       // We know addresses will be available in Firefox 56, even before
```

Comment:
@mhammond - will gladly do so for the upcoming train, I opened this PR against the current prod branch and want to keep the diff as small as possible.